scripts/run_mandiff.sh

```diff
@@ -20,7 +20,9 @@ WD=$(dirname "$0")
 WD=$(cd "$WD"; pwd)
 ROOT=$(dirname "$WD")
 OUT=${OUT:-/tmp/istio-mandiff-out}
-CHARTS_DIR="${GOPATH}/src/istio.io/installer"
+
+CHARTS_DIR=$(mktemp -d)
+git clone https://github.com/istio/installer.git "${CHARTS_DIR}"
```

Comment:
This is the root cause of `mandiff` test failure, because `helm template...` is using `istio/installer` master, while the `mesh manifest generate...` is using `istio/installer` with commit id from https://github.com/istio/operator/blob/master/installer.sha

Reply:
Thank you!